ts/src/pro/coinex.ts

```diff
@@ -21,22 +19,24 @@ export default class coinex extends coinexRest {
                 'watchTicker': true,
                 'watchTickers': true,
                 'watchTrades': true,
-                'watchMyTrades': false, // can query but can't subscribe
+                'watchMyTrades': true,
```

Comment:
I think we can also implement

`watchTradesForSymbols` and `watchOrderBookForSymbols`